activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -316,4 +316,5 @@ def set_defaults(target) # :doc:
   end
 end
 
+# :stopdoc:
 HashWithIndifferentAccess = ActiveSupport::HashWithIndifferentAccess
```

Comment:
Should not startdoc be below this line? Also do we want to remove it from the public API?